Dockerfile

```
@@ -0,0 +1,15 @@
+FROM alpine:3.10
```

```suggestion
FROM alpine:3.11
```

Much better but 3.11 is out so we should use that.